src/Components/ServiceScene/ActionBarScene.tsx

```diff
@@ -100,3 +106,66 @@ function getStyles(theme: GrafanaTheme2) {
     }),
   };
 }
+
+function ToolbarExtensionsRenderer(props: { serviceScene: SceneObject }) {
+  const [filters, setFilters] = useState<Array<{ key: string; value: string }>>(
+    getLabelsVariable(props.serviceScene).state.filters
+  );
+  useEffect(() => {
+    const sub = getLabelsVariable(props.serviceScene).subscribeToState((newState) => {
+      setFilters(newState.filters);
+    });
+    return () => {
+      sub.unsubscribe();
+    };
+  }, [props.serviceScene]);
+
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+  const extensions = usePluginLinks({
+    extensionPointId: 'grafana-lokiexplore-app/toolbar',
```

Comment:
Not sure about the naming. Considering the default text in dropdown states "open related" maybe "intent" should be part of the extension point name.